common/country_types/hfe_country_types_pirate.txt

```diff
@@ -21,14 +23,20 @@ hfe_pirate = {
 	enforces_borders = no
 	share_communications = no
 	attack_unarmed_orbital_stations = yes
-	can_have_their_comms_shared = no
+	can_have_their_comms_shared = yes
 	can_receive_envoys = no
 	mid_game_crisis = yes
 	sub_title = personality_hfe_personality_pirate
 	sub_title_desc = personality_hfe_personality_pirate_desc
+	pre_communications_fleet_name = RAMSHACKLE_FLEET
```

Comment:
It have instant communication

Reply:
Flavor is flavor. I was unaware of instant communications, TBH.